Source/Dafny/AST/DafnyAst.cs

```diff
@@ -2684,6 +2706,9 @@ public abstract class CollectionType : NonProxyType {
         return Arg.MayInvolveReferences;
       }
     }
+    public override bool DoesNotContainGhostConstraints() {
+      return Arg.DoesNotContainGhostConstraints();
```

Comment:
Is this covered in the tests?

Reply:
I haven't found a case when the ghost inference did consider a seq with a "ghost" subset type in its Range as "compilable", so I don't think I can come up with a case where returning true or false here would make a difference in files
Let's continue the discussion here:
https://github.com/dafny-lang/dafny/pull/1702#discussion_r846309761